apps/mobile/src/hooks/usePostActionSheet.js

```diff
@@ -57,14 +57,14 @@ export default function usePostActionSheet ({
   const [, pinPost] = useMutation(pinPostMutation)
   const { showHyloActionSheet } = useHyloActionSheet()
   const mixpanelTrack = useMixpanelTrack()
-  const [{ currentGroup }] = useCurrentGroup()
+  const [{ currentGroup, isContextGroupSlug }] = useCurrentGroup()
```

Comment:
I am thinking.... @lorenjohnson 

`isContextGroupSlug` should refer to the function; and we should change the boolean that pops out of useCurrentGroup as `isContextGroup`.

Two reasons; 1) using different symbols for related but different entities 2) `isContextGroup` is probably more semantically aligned when coming out of useCurrentGroup, since its all references to the current group

Reply:
yes for sure, go at it. it was just a bulk rename. definitely should be isContextGroup here, however my feeling here is that we should just promote that up to the now almost blank GroupPresenter as an isStatic property, or something similar that will translate coherently also in Web. In Mobile, "Context Group" meant a group that was statically defined and only there to create a context. I think that groups to us have finally entirely become understood as contexts and so the only thing left to delineate these previously "context groups" in mobile would be i think most sensibly staticGroups or systemGroups, so isStatic is my vote...